disaggregated_memory/configs/arm-main.py

```diff
@@ -0,0 +1,299 @@
+# Copyright (c) 2023-24 The Regents of the University of California
```

Comment:
WHy is this file separate from riscv-main.py? The only difference is the board and the workload. I think you could factor out all of the common code!

Reply:
I can do that. I'll update this and push this in the next patch.

Reply:
Actually, we should keep these configs separate imo. Otherwise, the user has to compile ALL/gem5. We can discuss more on this.